packages/format-library/src/image/index.js

```diff
@@ -73,24 +77,43 @@ function InlineUI( { value, onChange, activeObjectAttributes, contentRef } ) {
 						replacements: newReplacements,
 					} );
 
+					setHasChanged( false );
+
 					event.preventDefault();
 				} }
 			>
-				<HStack alignment="bottom">
+				<VStack>
 					<NumberControl
 						label={ __( 'Width' ) }
 						value={ width }
 						min={ 1 }
-						onChange={ ( newWidth ) => setWidth( newWidth ) }
-						size="__unstable-large"
+						onChange={ ( newWidth ) => {
+							setWidth( newWidth );
+							setHasChanged( true );
+						} }
+						size="compact"
 					/>
-					<Button
-						icon={ keyboardReturn }
-						label={ __( 'Apply' ) }
-						type="submit"
-						__next40pxDefaultSize
+					<TextareaControl
+						label={ __( 'Alternative text' ) }
+						__nextHasNoMarginBottom
+						value={ alt }
+						onChange={ ( newAlt ) => {
+							setAlt( newAlt );
+							setHasChanged( true );
```

Comment:
Why is this needed? Can't we compare the current state?

Reply:
That's true. I removed the redundant state by cf66b8e. The approach is similar to [this one](https://github.com/WordPress/gutenberg/blob/d5e7a77eb851329da16510f758bdd742099fb5ac/packages/block-editor/src/components/block-rename/modal.js#L29-L31).

If there is a change from the initial value, the Apply button should become active:

https://github.com/user-attachments/assets/b3c695b0-4334-421a-a91b-5b67a72cbf32